opt/connectedcars/package-json-rewrite/package-json-rewrite.js

```diff
@@ -1,4 +1,10 @@
-const githubRegex = /^git\+ssh:\/\/git(@github.com)/
+const providers = {
+    github: 'github.com'
+}
+
+const providerRegex = new RegExp(
+    `^(?<provider>${Object.keys(providers).join('|')}):(?<username>.*)/(?<repository>[^#]*)(?<version>#.+)?$`
+  )
```

Comment:
What's with the random space here?

Reply:
I don't have any linting config for this repo because it's mainly docker, I think I will move it into its own repo so we can use it in other builder images as well and also get linting.